### PR TITLE
Landcover=*

### LINKED ISSUE #6: Display landcover
It would be nice to display

- landcover = grass (the same as the current landuse=grass)
- landcover = trees (the same as the current landuse=forest)
- landcover = bushes (the same as the current natural=scrub)
- landcover = sand (the same as the current natural=sand)

those are the most important to me. See https://wiki.openstreetmap.org/wiki/Proposed_features/landcover for other values.

### PR DESCRIPTION
**Pull request by pietervdvn**
I like landcovers more then some landuses!

Closes #6 ; let's try it out!

### CODE REVIEW

**Comment by jbelien** [comment]
Hey @pietervdvn ,

Thanks for trying this out !
Obviously, I won't merge it right away ; let's first see what people think about it :)

Could you first resolve the conflicting file : `landcover.mss` ?
Thanks !

**Review by jbelien — CHANGES_REQUESTED** [review]
Could you first resolve the conflicting file : `landcover.mss` ?

**Comment by pietervdvn** [comment]
The conflict has been resolved

**Comment by jbelien** [comment]
Could you also make sure `openstreetmap-carto.lua` and `openstreetmap-carto.style` with the necessary tags ?

File: landcover.mss [hunk 1]
```diff
@@ -340,13 +341,18 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass']
   [feature = 'landuse_village_green'][zoom >= 10],
-  [feature = 'leisure_common'][zoom >= 10] {
+  [feature = 'leisure_common'][zoom >= 10]
```
Comment (jbelien) [inline landcover.mss]: Could you keep the syntax as the rest of the file for consistency ?
```
[feature = 'leisure_common'][zoom >= 10] {
```

File: landcover.mss [hunk 2]
```diff
@@ -340,13 +341,18 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass']
```
Comment (jbelien) [inline landcover.mss]: A comma `,` is missing at the end of the line.

File: landcover.mss [hunk 3]
```diff
@@ -340,13 +341,18 @@
...
+  [feature = 'leisure_common'][zoom >= 10]
+ {
     polygon-fill: @grass;
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+
+
+
```
Comment (jbelien) [inline landcover.mss]: No need for these new lines.

File: landcover.mss [hunk 4]
```diff
@@ -476,7 +482,9 @@
     }
   }
 
-  [feature = 'natural_sand'][zoom >= 8] {
+  [feature = 'natural_sand'][zoom >= 8],
+  [feature = 'landcover_sand'][zoom >= 8]
```
Comment (jbelien) [inline landcover.mss]: Could you keep the syntax as the rest of the file for consistency ?
```
[feature = 'landcover_sand'][zoom >= 8] {
```

File: landcover.mss [hunk 5]
```diff
@@ -488,7 +496,9 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
-  [feature = 'natural_scrub'][zoom >= 10] {
+  [feature = 'natural_scrub'][zoom >= 10],
+  [feature = 'landcover_bushes'][zoom >= 10]
```
Comment (jbelien) [inline landcover.mss]: Could you keep the syntax as the rest of the file for consistency ?
```
[feature = 'landcover_bushes'][zoom >= 10] {
```

**Review by jbelien — CHANGES_REQUESTED** [review]

File: landcover.mss (hunk 1 above)
Comment (pietervdvn) [reply landcover.mss]: Fixed

File: landcover.mss (hunk 2 above)
Comment (pietervdvn) [reply landcover.mss]: Fixed

File: landcover.mss (hunk 3 above)
Comment (pietervdvn) [reply landcover.mss]: Fixed

**Comment by jbelien** [comment]
Seems good ! Thanks @pietervdvn !

I'll keep you updated when it's available (I would say first week of April at best).

**Comment by pietervdvn** [comment]
I guess the next sunday batch? That was what I was expecting, so...

Thanks for considering!


Met vriendelijke groeten,
Pieter Vander Vennet

2018-03-26 15:48 GMT+02:00 Jonathan Beliën <notifications@github.com>:

File: landcover.mss [hunk 6]
```diff
@@ -340,6 +341,7 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass'],
```
Comment (jbelien) [inline landcover.mss]: I think there is a `[zoom >= 10]` missing here...

Comment (jbelien) [inline landcover.mss]: I think there is a `[zoom >= 10]` missing here...

**Review by jbelien — CHANGES_REQUESTED** [review]

File: landcover.mss (hunk 6 above)
Comment (marcgemis) [reply landcover.mss]: I agree

Comment (pietervdvn) [reply landcover.mss]: Indeed, fixed.

**Review by jbelien — APPROVED** [review]

**Comment by jbelien** [comment]
Not sure exactly what I did that closed that branch automatically (I guess it was the branch renaming).

Following the work done with Champs-Libres to improve our OSMBE tileserver, I decided to simplify the `openstreetmap-carto-be` repository.
I'll keep here only the required files in a branch with the `openstreetmap-carto` release number.

I think this PR is still relevant. Could you create a new PR with only the necessary files and modifications to branch `5.1` ? Thanks :)

**Pull request closed without merging**